UnityProject/Assets/Scripts/Core/Physics/UniversalObjectPhysics.Rotation.cs

```diff
@@ -6,45 +6,19 @@ namespace Core.Physics
 	public partial class UniversalObjectPhysics
 	{
 		protected Rotatable rotatable;
-		private Vector3 initialRotationOnAwake = Vector3.zero;
 		[SerializeField] protected Transform rotationTarget;
 
-		private void SetInitialRotationTarget()
-		{
-			if (rotationTarget != null) return;
-			initialRotationOnAwake = rotationTarget.rotation.eulerAngles;
-		}
 
 		private void SetRotationTarget()
 		{
-			if (rotationTarget != null) return;
-			if (this is MovementSynchronisation c && c.playerScript.RegisterPlayer.LayDownBehavior != null)
```

Comment:
these are all fixes for other problems
revert this if we start seeing rotation problems again next playtest.